MacOSLinuxInstall.md

```diff
@@ -0,0 +1,123 @@
+
+eval "$(pyenv init -)"
+
+```
+
+4. Close your terminal and then open it again. You won't notice any changes yet, but we have just made it possible to use different versions of Python through PyEnv without removing your current version.
+
+Proceed to Configuring and Running PyEnv and osr2mp4 to continue.
+
+-------------------------------------------------------------------------------------------------------------------------------------
+
+## MacOS
+
+1. The easiest way to use osr2mp4 on MacOS will be by installing most of our components with Brew, so please make sure you have this configured before we continue. MacOS has the following requirements before we can begin the installation process: `brew install zlib jpeg freetype xz ffmpeg`. After these are installed we can begin. Install PyEnv through Brew: `brew install pyenv`.
+
+2. Add `pyenv-init` to your shell configuration file (~/.bash_profile in MacOS). The easiest way to do this is with your text editor of choice. Open the file and a new line `eval "$(pyenv init -)"` at the end. It is important that this is put at the end of the file in order to prevent unwanted behavior in your terminal.
+
+3. Close your terminal and then open it again. You won't notice any changes yet, but we have just made it possible to use different versions of Python through PyEnv without removing your current version.
+
+Proceed to Configuring and Running PyEnv and osr2mp4 to continue.
+
+-------------------------------------------------------------------------------------------------------------------------------------
+
+# Configuring and Running PyEnv and osr2mp4
+
+1. Verify that PyEnv has been properly installed by typing `pyenv --version`. It should return a version number. If not, check the output from your installation and see if there are errors.
+
+2. Close your terminal and then open it again. You won't notice any changes yet, but we have just made it possible to use different versions of Python through PyEnv without removing your current version.
+
+3. Install Python 3.6.X. Use `pyenv versions` to list all of the versions available for install. In this example we will be using Python 3.6.8: `pyenv install 3.6.8`. This may take a long time depending on how powerful your processor is.
+
+**Before you proceed, you should switch to Python 3.6 in your terminal or nothing will work.** Do this with `pyenv shell 3.6.8` We're now using Python 3.6.8 in our Terminal. Great! But before we proceed we need to upgrade pip, Python's package manager. `pip install --upgrade pip`.
+
+4. Now change your directory to wherever you need want to install osr2mp4 and clone the osr2mp4 repository, then step into the osr2mp4-app folder in your terminal. This should look something like this:
+
+```
+
+          ~ $ cd /home/username/programs
+
+   programs $ git clone https://github.com/uyitroa/osr2mp4-app
+
+   programs $ cd osr2mp4-app
+
+```
+
+6. Now install dependencies for osr2mp4: `pip install -r requirements.txt`.
+
```

Comment:
I don't remember if python comes with library Pillow by default, if so then the user needs to uninstall PIL and Pillow before installing Pillow-SIMD

Reply:
I didn't have to install them but some people might. I'll add that and fix the other issue this evening.

Reply:
Yes you don't have to uninstall it, but the app will then use Pillow, which is slower than Pillow-SIMD.